easy_thumbnails/conf.py

```diff
@@ -302,7 +299,7 @@ class Settings(AppSettings):
         {% thumbnail obj.image 50x50 crop HIGH_RESOLUTION %}  {# force hires #}
     """
 
-    THUMBNAIL_HIGHRES_INFIX = '@2x'
+    THUMBNAIL_HIGHRES_INFIX = "@2x"
```

Comment:
I think we can drop that hack altogether. It was introduced in 2013, because back then there was no browser support for Retina images. Nowadays this can be achieved in a much cleaner way.

Reply:
That is somthing more that style fixes, I think it’s better to do it separately.